snap/snapcraft.yaml

```
@@ -1,71 +1,72 @@
+
+  We heard from users who record podcasts with our multi-channel audio recorder, players seeking realism with our positional audio in games, Eve Online players with huge communities of over 100 simultaneous voice participants (I bet they make good use of our extensive permission system 😄), the competitive Team Fortress 2 community making us their required voice communication platform, hobby radio transmission users, and a variety of workplaces adapting Mumble to fit their needs - be it on-head mobile devices or communicating across countries or into airplanes.
+
+  Administrators appreciate Mumble for being able to self host and have control over data security and privacy. Some make use of the extensive permission system for complex scenarios (for example separating two groups but leaders being able to talk to both). Some love to provide their users with additional functionality with scripts making use of server APIs, or host music bots and the like that connect to the server. Those that have an existing user database often make use of authenticators to allow authenticating with existing account login data.
+
+  **Features for end-users**
+
+  - Low-latency - great for talking and gaming
+  - Stay private and secure: always encrypted communication and public/private-key authentication by default.
+  - Recognize friends across servers
+  - For gamers: In-game Overlay - see who is talking, FPS and the current time
+  - For gamers: Positional audio - hear the players from where they are located in-game
+  - Wizards to guide you through setup, like configuring your microphone
+
+  **Features for administrators**
+
+  - Libre software - no licensing hassle, caveats and limitations
+  - Open Source - open in security and technology, and open to extendibility
+  - Extensive user permission system (ACL)
+  - Extendible through Ice protocols
+  - Web interfaces - free choice of several community free software projects
+  - Channel viewers - even without direct Ice access, if the hoster provides the CVP, easily set up your channel viewer of choice
+  - Authenticators - to allow users to authenticate against an existing user database
+  - Custom chat commands and context (right click) menu entries
+
+  **Authors**
+
+  This snap is maintained by the Snapcrafters community, and is not endorsed or officially maintained by the upstream developers.
+website: https://www.mumble.info/
+contact: https://github.com//snapcrafters/mumble/issues
+issues: https://github.com//snapcrafters/mumble/issues
+source-code: https://github.com//snapcrafters/mumble
+icon: img/mumble-logo.png
+license: BSD-3-Clause
 adopt-info: mumble
-base: core18
 
+base: core20
 grade: stable
 confinement: strict
 
 apps:
   mumble:
-    command: bin/mumble
-    command-chain: [bin/desktop-launch]
+    command: usr/local/bin/mumble
```

I prefer to use `usr` rather than `usr/local` for installs to mimic a native package more closely inside the snap filesystem. This means in this case using `-DCMAKE_INSTALL_PREFIX=/usr` in the part below.

```suggestion
    command: usr/bin/mumble
```